### ISSUE TITLE
Bad results with eval.py

### ISSUE THREAD

**Issue by ArmandoDomi** [GuillaumeMougeot/DogFaceNet#3]
During training a get val_triplet_acc 85-90 % and when i use the eval.py the prediction are random ( 48-52 % accuracy). i don't understand how this is possible?. I am using the lfw dataset

**Comment by GuillaumeMougeot** [comment]
Sorry to read that... DogFaceNet repo is unfortunately no longer maintained. I don't know where this problem could come from... but I do have ideas. I had many problems with the early version of Keras at that time. I tried to re-implement this work using tensorflow 2.0 + keras when they came out and I found many unseen bugs in my previous work. Notably, the model used here is far to be the best and the loss definition also (though FaceNet-like losses where still better than angle-wise losses like ArcFace). These updates are not yet published and I'm not sure they will be. The current DogFaceNet repo is here more to illustrate the method presented in the paper and to store the dataset than to be actually used... Sorry again
